src/main/java/com/dev/cinema/service/impl/mapper/MovieSessionMapper.java

```diff
@@ -0,0 +1,45 @@
+package com.dev.cinema.service.impl.mapper;
+
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.dto.MovieSessionRequestDto;
+import com.dev.cinema.model.dto.MovieSessionResponseDto;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.MovieService;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapper {
+    private final CinemaHallService cinemaHallService;
+    private final MovieService movieService;
+
+    @Autowired
+    public MovieSessionMapper(CinemaHallService cinemaHallService, MovieService movieService) {
+        this.cinemaHallService = cinemaHallService;
+        this.movieService = movieService;
+    }
+
+    public MovieSession toEntity(MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession movieSession = new MovieSession();
+        movieSession.setId(movieSessionRequestDto.getMovieId());
```

Comment:
is this line ok?

Reply:
id is always null? removed it

Reply:
I have just rewritten my class and removed the cinemaHallCapacity variable